eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -22,14 +22,39 @@
   <!-- Public API already released with incorrect upper-case naming pattern -->
   <suppress checks="MethodName" files="AsymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="KeyWrapAlgorithm.java"/>
+  <suppress checks="MethodName" files="PartitionSenderImpl.java"/>
   <suppress checks="MethodName" files="SignatureEncoding.java"/>
   <suppress checks="MethodName" files="SymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="RsaSignature.java"/>
 
   <!-- Public API already released with incorrect constant variable naming -->
   <suppress checks="ConstantName" files="AlgorithmResolver.java"/>
+  <suppress checks="ConstantName" files="AmqpErrorCode.java"/>
+  <suppress checks="ConstantName" files="BatchErrorCodeStrings.java"/>
+  <suppress checks="ConstantName" files="ConnectionStringBuilder.java"/>
+  <suppress checks="ConstantName" files="EventProcessorHost.java"/>
+  <suppress checks="ConstantName" files="InMemoryCheckpointManager.java"/>
+  <suppress checks="ConstantName" files="InMemoryLeaseManager.java"/>
+  <suppress checks="ConstantName" files="MessagingFactory.java"/>
+  <suppress checks="ConstantName" files="PartitionManagerOptions.java"/>
+  <suppress checks="ConstantName" files="PartitionScanner.java"/>
   <suppress checks="ConstantName" files="RsaKey.java"/>
   <suppress checks="ConstantName" files="SymmetricKey.java"/>
-  <suppress checks="ConstantName" files="BatchErrorCodeStrings.java"/>
   <suppress checks="ConstantName" files="TaskFailureInformationCodes.java"/>
+
+  <!-- Public API already released with incorrect static variable naming -->
+  <suppress checks="StaticVariableName" files="EventHubClientImpl.java"/>
+
+  <!-- Public API already released with visibility modifier -->
+  <suppress checks="VisibilityModifier" files="BatchOptions.java"/>
+  <suppress checks="VisibilityModifier" files="EventHubClientImpl.java"/>
+
+  <!-- Public API already released with inner assignment-->
+  <suppress checks="InnerAssignment" files="MessageReceiver.java"/>
+
+  <!-- Public API already released with empty block-->
+  <suppress checks="EmptyBlock" files="MessageReceiver.java"/>
+  <suppress checks="EmptyBlock" files="InMemoryLeaseManager.java"/>
+  <suppress checks="EmptyBlock" files="ReactorDispatcher.java"/>
+  <suppress checks="EmptyBlock" files="AzureStorageCheckpointLeaseManager.java"/>
```

Comment:
I think this could be fixed.
```java
if (wasLeaseLost(se, azLease.getPartitionId())) {
    // leave result as false
} else {
  	throw se;
}
```
could be
```java
if (!wasLeaseLost(se, azLease.getPartitionId())) {
    throw se;
}
// otherwise, leave result as false.
```